depends/protobuf/CMakeLists.txt

```diff
@@ -202,6 +202,9 @@ LIST(APPEND LIBPROTOBUF_FULL_SRCS ${LIBPROTOBUF_LITE_SRCS})
 IF(CMAKE_COMPILER_IS_GNUCC)
     SET(CMAKE_CXX_FLAGS_RELWITHDEBINFO "${CMAKE_CXX_FLAGS_RELWITHDEBINFO} -Wno-sign-compare")
     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-unused-result")
+ELSEIF(MSVC)
+    # Disable warnings for integer conversion to smaller type
+    SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /wd4267")

```

Comment:
@lethosor, it's protobuf-only CMakeLists.txt, it's probably fine to suppress the warnings in this case rather than fix anything in 3rd-party code.